src/r2x/parser/plexos.py

```diff
@@ -168,20 +169,20 @@ def __init__(self, *args, xml_file: str | None = None, **kwargs) -> None:
         # TODO(pesap): Rename exceptions to include R2X
         # https://github.com/NREL/R2X/issues/5
         # R2X needs at least one of this maps defined to correctly work.
-        if (
-            not self.fuel_map
-            and not self.device_map
-            and not self.device_match_string
-            and not self.category_map
-        ):
-            msg = (
-                "Neither `plexos_fuel_map` or `plexos_device_map` or `device_match_string` was provided. "
-                "To fix, provide any of the mappings."
-            )
+        one_required = ["fuel_map", "device_map", "device_match_string", "category_map"]
+        if all(getattr(self, one_req, True) for one_req in one_required):
+            msg = f'At least one of {", or ".join(one_required)} is required to initialize PlexosParser'
```

Comment:
Can you add testing to verify that we raise this error?

Reply:
will do

Reply:
should be added now